content/docs/getting-started/fluvio-local-faq.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Troubleshoot Fluvio Installation
+menu: Troubleshooting
+toc: true
+weight: 40
+---
+
+Fluvio is a very young project, so we're still ironing out all the wrinkles
+when it comes to the first-time installation process. If you get stuck while
+following the [Installing Fluvio Locally] guide, check this page to see if
+you're running into one of the more common errors. If you still can't figure
+it out, please feel free to [open an issue] or to drop by
+[our Discord channel] to ask us for help!
+
+[Installing Fluvio Locally]: ./fluvio-local.md
+[open an issue]: https://github.com/infinyon/fluvio/issues/new
+[our Discord channel]: https://discord.gg/Z6ebmmh
+
+## Help, I'm stuck
+
+Let's try to get you unstuck. Here are some known installation problems and
+how to fix them. We're working to shrink this list over time :)
+
+### `fluvio cluster install`: Fluvio system chart is not installed
+
+If you see an error like the one below, it's likely that you forgot to run
+the install command with the `--sys` option first.
+
+```
+$ fluvio cluster install --chart-version=0.6.0-latest --image-version=latest
+Error: 
```

Comment:
We should not display raw error message from Rust.  Please open ticket for Fluvio to convert to better readable message.